worktrolly-ui/src/app/body/task-details/log-work/log-work.component.ts

```diff
@@ -15,14 +16,17 @@ export class LogWorkComponent implements OnInit {
   @Input('task') task: Tasks
   @Output() logWorkCompleted = new EventEmitter<{ completed: boolean }>();
 
+  logWork: string= "/ Log Work"
+  taskDetails: string= "/ Task Details"
```

Comment:
/ can be handled in navbar component.

Reply:
Also give name to each component, like componentName = "abc".

And send this component name to navbar component.

Then in navbar component have an array of component names. That can be shown to the navbar component.
The component which is at the end will only be marked green.
Also perform push and pop operation in navbar component over the list of components that are in the arraylist